src/js/components/AppListComponent.jsx

```diff
@@ -168,17 +168,30 @@ var AppListComponent = React.createClass({
   },
 
   hasFilters: function () {
-    return Object.values(this.props.filters).some(filter => {
-      return filter != null &&
-        (Util.isArray(filter) && filter.length > 0) ||
-        (Util.isString(filter) && filter !== "");
+    var filters = this.getQueryParamObject();
+
+    filters = Object.keys(filters).filter(key => {
```

Comment:
Great! Thanx for fixing my ****.

Slightly improved versinon. This approach has a max iteration of values defined in `FilterTypes`.

```javascript
    filters = Object.values(FilterTypes)
      .reduce((returnObject, key) => {
        var filter = filters[key];
        if (filter != null) {
          returnObject[key] = filter;
        }
        return returnObject;
      }, {});
```